web-app/src/main/webapp/index.html

```diff
@@ -19,8 +20,68 @@
         </div>
       </form>
     </div>
+    <!-- Map -->
```

Comment:
Not sure if html comments are necessary since it is pretty self-explanatory from the id and class names.

Reply:
I'm in favour of keeping them because the html just get's really long later on and comments seem like a good way to visually differentiate the sections, but I'm not sure if it's discouraged.